config/src/test/groovy/org/springframework/security/config/annotation/web/configurers/CsrfConfigurerTests.groovy

```diff
@@ -507,6 +509,108 @@ class CsrfConfigurerTests extends BaseSpringSpec {
 		}
 	}
 
+	def 'ignoring request matchers augment configured request matcher'() {
```

Comment:
Please do not create new Groovy tests. Instead you can create another Java based test with the new tests in it. Sometimes I will rename the old Groovy tests to end in Spec to avoid duplicate classes with the same name.

This comment should be applied generally to all code going into Spring Security (i.e. make this change for all .groovy files that were edited in this PR)

Reply:
Ah, okay, sure. I can move these easily to their Java equivalent.

Reply:
I see that there is a class called `CsrfConfigurerNoWebMvcTests`. Was going to create `CsrfConfigurerJavaTests` (to differentiate from the groovy `CsrfConfigurerTests`) before I saw this.

What convention should I be shooting for? For example, is naming it `CsrfConfigurerRequestMatcherTests` more appropriate?